contracts/UFragmentsPolicy.sol

```diff
@@ -203,6 +238,16 @@ contract UFragmentsPolicy is Ownable {
 
         uFrags = uFrags_;
         baseCpi = baseCpi_;
+
+        rebaseTimeSec = 82800;  // 11PM UTC 82800
+        rebaseTimeWindowSizeSec = 15 minutes;
+    }
+
+    function isWithinRebaseWindow() public view returns (bool) {
+        return (
+            (now % minRebaseTimeIntervalSec) > rebaseTimeSec &&
+            (now % minRebaseTimeIntervalSec) <= (rebaseTimeSec + rebaseTimeWindowSizeSec)
```

Comment:
```suggestion
            (now % minRebaseTimeIntervalSec) < (rebaseTimeSec + rebaseTimeWindowSizeSec)
```

Reply:
The way I am thinking about it: If rebaseTimeSec = 0 and rebaseTimeWindowSizeSec = minRebaseTimeIntervalSec then this method should always return true

Reply:
Agreed 👍 I actually did endup making this change locally when I was writing the tests